recipe/meta.yaml

```diff
@@ -23,7 +23,7 @@ requirements:
   run:
     - attrs >=17.4.0
     # TODO: upstream, this is conditional to <3.8, but probably not worth the non-noarch build
-    - importlib_metadata
+    - importlib-metadata
```

Comment:
In theory it should not matter b/c one is a shim to the other... But... Yeah, we should depend on the canonical one.